flake.nix

```diff
@@ -9,15 +9,22 @@
     lib = {
       hm = import ./modules/lib { lib = nixpkgs.lib; };
       homeManagerConfiguration = { configuration, system, homeDirectory
-        , username, extraSpecialArgs ? { }
+        , username, stateVersion ? "20.09", extraModules ? [ ]
```

Comment:
Considering that `stateVersion` defaults to the version of the time it was introduced, I think it's safe to use a state version here too, though I would change it to `21.05`.

cc @rycee

Reply:
I put it as a minimum version that is required and thought it was okay to default to that. But if we think it is better to default to the latest and let people with older state version set it I guess that is fine as well?

Reply:
I think we should set the `stateVersion` to `null` here and open a new PR where we set the default `stateVersion` in `modules/misc/version.nix` to be the version of the latest release.